src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs

```diff
@@ -623,151 +625,105 @@ public async Task DecompressArchive()
         {
             BaseStorageFile archive = await StorageHelpers.ToStorageItem<BaseStorageFile>(associatedInstance.SlimContentPage.SelectedItem.ItemPath);
 
-            if (archive != null)
+            if (archive == null)
             {
-                DecompressArchiveDialog decompressArchiveDialog = new();
-                DecompressArchiveDialogViewModel decompressArchiveViewModel = new(archive);
-                decompressArchiveDialog.ViewModel = decompressArchiveViewModel;
-
-                ContentDialogResult option = await decompressArchiveDialog.ShowAsync();
-
-                if (option == ContentDialogResult.Primary)
-                {
-                    // Check if archive still exists
-                    if (!StorageHelpers.Exists(archive.Path))
-                    {
-                        return;
-                    }
+                return;
+            }
 
-                    CancellationTokenSource extractCancellation = new();
-                    PostedStatusBanner banner = App.OngoingTasksViewModel.PostOperationBanner(
-                        archive.Name.Length >= 30 ? archive.Name + "\n" : archive.Name,
-                        "ExtractingArchiveText".GetLocalizedResource(),
-                        0,
-                        ReturnResult.InProgress,
-                        FileOperationType.Extract,
-                        extractCancellation);
+            DecompressArchiveDialog decompressArchiveDialog = new();
+            DecompressArchiveDialogViewModel decompressArchiveViewModel = new(archive);
+            decompressArchiveDialog.ViewModel = decompressArchiveViewModel;
 
-                    BaseStorageFolder destinationFolder = decompressArchiveViewModel.DestinationFolder;
-                    string destinationFolderPath = decompressArchiveViewModel.DestinationFolderPath;
+            ContentDialogResult option = await decompressArchiveDialog.ShowAsync();
 
-                    if (destinationFolder == null)
-                    {
-                        BaseStorageFolder parentFolder = await StorageHelpers.ToStorageItem<BaseStorageFolder>(Path.GetDirectoryName(archive.Path));
-                        destinationFolder = await FilesystemTasks.Wrap(() => parentFolder.CreateFolderAsync(Path.GetFileName(destinationFolderPath), CreationCollisionOption.GenerateUniqueName).AsTask());
-                    }
-                    if (destinationFolder == null)
-                    {
-                        return; // Could not create dest folder
-                    }
+            if (option != ContentDialogResult.Primary)
+            {
+                return;
+            }
```

Comment:
```suggestion
            if (option != ContentDialogResult.Primary)
                return;
```

Brackets are only used on multiline if statements

Reply:
I could've sworn the style guide said brackets for all. Must've of been a different one. Updated.

Reply:
Style guide was recently updated 🙂